src/Microsoft.AspNet.Authentication/CertificateSubjectKeyIdentifierValidator.cs

```diff
@@ -27,7 +27,7 @@ public CertificateSubjectKeyIdentifierValidator([NotNull] IEnumerable<string> va
 
             if (_validSubjectKeyIdentifiers.Count == 0)
             {
-                throw new ArgumentOutOfRangeException("validSubjectKeyIdentifiers");
+                throw new ArgumentOutOfRangeException(nameof(_validSubjectKeyIdentifiers));
```

Comment:
@Tratcher should this be a more general `ArgumentException` since the provided `string` isn't out of anybody's range?